virttest/vt_cluster/__init__.py

```
@@ -0,0 +1,315 @@
+    def __init__(self):
+        self._filename = os.path.join(data_dir.get_base_backend_dir(), "cluster_env")
+        self._empty_data = {
+            "partitions": [],
+            "nodes": {},
+        }
+        try:
+            if os.path.isfile(self._filename):
+                self._data = self._load()
+            else:
+                self._data = self._empty_data.copy()
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)